packages/components/psammead-inline-link/README.md

```diff
@@ -0,0 +1,92 @@
+
+## Description
+
+The `@bbc/psammead-inline-link` package exports a single Inline Link component. It uses an `a` HTML element and colours from `@bbc/psammead-styles`.
+
+## Installation
+
+```
+npm install @bbc/psammead-inline-link --save
+```
+
+## Usage
+
+```jsx
+import InlineLink from '@bbc/psammead-inline-link';
+
+const WrappingComponent = () => (
+  <InlineLink href="https://www.bbc.com/news">Text here</InlineLink>
+);
+```
+
+## When to use this component
+
+This component can be used at any point on a page.
+
+We have not included typography styles in with this component. This is so it is resusable in multiple contexts. You can use our existing `psammead-paragraph` for body copy and `psammead-headings` for headlines and subheadings (`h1`s and `h2`s), in this way:
+
+In a paragraph:
+
+```jsx
+import InlineLink from '@bbc/psammead-inline-link';
+import Paragraph from '@bbc/psammead-paragraph';
+
+const WrapperComponent = () => (
+  <Paragraph>I am a paragraph that has a link to <InlineLink href="https://www.bbc.com/news">the BBC News front page</InlineLink>.<Paragraph>
+);
+```
+
+In a Headline or SubHeading:
+
+```jsx
+import InlineLink from '@bbc/psammead-inline-link';
+import { Headline, SubHeading } from '@bbc/psammead-headings';
+
+const HeadlineWithALink = () => (
+  <Headline><InlineLink href="https://www.bbc.com/news">Headline</InlineLink>
+  <Headline>
+
+  const SubHeadingWithALink = () => (
+  <SubHeading><InlineLink href="https://www.bbc.com/news">SubHeading</InlineLink><SubHeading>
+);
+```
+
+Alternatively, if you want to just extend existing styles with other GEL Typography groups, you can do the following:
+
+```jsx
+import InlineLink from '@bbc/psammead-inline-link';
+import { GEL_PARAGON } from '@bbc/gel-foundations-styled-components';
+
+const GelParagonLink = styled(InlineLink)`
+  ${GEL_PARAGON};
+`;
+
+const WrapperComponent = () => (
+  <GelParagonLink href="https://www.bbc.com/news">
+    Link with GEL Paragon typography
+  </GelParagonLink>
+);
+```
+
+## Accessibility notes
+
+Since this is just a `<a>` tag with associated styles, when you use this component, it has the same semantic meaning as a regular anchor element.
+
+The font and background-color choices for each hover/focused/visited/default states meet WCAG AA colour contrast guidelines.
+
+## Roadmap
+
+Our UX team are considering updating the colours for the hover and focus states of this component. When we implement this change, we will ensure it is a major version increase, since for downstream teams this will be a breaking change.
```

Comment:
Unless the API changes... if this is just color tweaks etc, I'd maybe advocate a minor change 🤔

Reply:
As a consumer of this component - imagine if you're using `psammead-inline-link` in production code and the next time you rebuild your application and deploy it, all of your link colours automatically update. That is a quick way to get a users of `psammead` components frustrated with us! 

So, although I understand that colour tweaks are usually just a patch or a minor change in a standard component, when you're a component library being used - potentially - by multiple teams, we need to consider even colours and styles as part of our API.